presto-parquet/src/main/java/io/prestosql/parquet/reader/ParquetReader.java

```diff
@@ -149,6 +150,7 @@ private boolean advanceToNextRowGroup()
         currentRowGroupMemoryContext.close();
         currentRowGroupMemoryContext = systemMemoryContext.newAggregatedMemoryContext();
 
+        freeChunkReaders(currentRowGroup, columns.size());
```

Comment:
maybe rename this to something like `freeCurrentRowGroupBuffers`, and reference the fields directly